File: src/GraphRAG.Code/Agency.GraphRAG.Code.Test/Chunker/Fixtures/typescript/Simple.ts
import { format } from "node:util";

export class Worker {
  async run(task: string): Promise<string> {
    const parts = [task, format("done")];
    const message = parts.join(":");
    return message.trim();
  }
}

export function buildMessage(name: string): string {
  const greeting = `hello ${name}`;
  return greeting.toUpperCase();
}

export const formatUser = (name: string): string => {
  const normalized = name.trim();
  return normalized.toLowerCase();
};

File: src/GraphRAG.Code/Agency.GraphRAG.Code.Test/Chunker/Fixtures/python/simple.py
import os
from collections import deque


class Worker:
    def run(self, value: int) -> str:
        return str(value)

    def stop(self) -> None:
        return None


def build_name(name: str) -> str:
    return name.upper()


def create_queue() -> deque[str]:
    return deque()
